### DISCUSSION TITLE
Scaricamento massivo pagelle

### DISCUSSION THREAD

**Discussion by sempliciniluigi89** [iisgiua/giuaschool#526] [Idee e proposte]
Buonasera, finora nel registro è possibile scaricare dal lato dirigente/segreteria le pagelle cliccando su alunno per alunno ,mi chiedo invece se esiste un modo per scaricare massivamente per classe

**Comment by trinko** [comment]
No, in effetti non esiste. Lo segno come richiesta
La segno come richiesta.
